types/async/index.d.ts

```diff
@@ -28,7 +28,7 @@ interface AsyncQueue<T> {
     idle(): boolean;
     concurrency: number;
     push<E>(task: T, callback?: ErrorCallback<E>): void;
-    push<E>(task: T, callback?: AsyncResultCallback<T, E>): void;
+    push<R,E>(task: T, callback?: AsyncResultCallback<T, R>): void;
```

Comment:
I made a mistake. Acutally it has to be `AsyncResult<E,R>`. I will correct it right now.